lib/updateInd.ml

```diff
@@ -480,18 +480,127 @@ and eval_person_var (fn, sn, oc, create, _) = function
+      match c with
+      | Update.Create (_, Some { ci_birth_place = pl }) ->
+          safe_val (Util.escape_html pl :> Adef.safe_string)
+      | _ -> str_val "")
+  | "birth_year" -> (
+      str_val
+      @@
+      match c with
+      | Update.Create (_, Some ci) -> (
+          match ci.ci_birth_date with
+          | Some (Dgreg (dmy, Dfrench)) ->
+              let dmy = Calendar.french_of_gregorian dmy in
+              add_precision (string_of_int dmy.year) dmy.prec
+          | Some (Dgreg ({ year = y; prec = p }, _)) ->
+              add_precision (string_of_int y) p
+          | Some _ -> ""
+          | None -> if ci.ci_public then "p" else "")
+      | _ -> "")
+  | "death_day" -> (
+      str_val
+      @@
+      match c with
+      | Update.Create (_, Some { ci_death_date = Some (Dgreg (dmy, Dfrench)) })
+        ->
+          let dmy = Calendar.french_of_gregorian dmy in
+          if dmy.day <> 0 then string_of_int dmy.day else ""
+      | Update.Create (_, Some { ci_death_date = Some (Dgreg ({ day = d }, _)) })
+        when d <> 0 ->
+          string_of_int d
+      | _ -> "")
+  | "death_month" -> (
+      str_val
+      @@
+      match c with
+      | Update.Create (_, Some { ci_death_date = Some (Dgreg (dmy, Dfrench)) })
+        ->
+          let dmy = Calendar.french_of_gregorian dmy in
+          short_f_month dmy.month
+      | Update.Create
+          (_, Some { ci_death_date = Some (Dgreg ({ month = m }, _)) })
+        when m <> 0 ->
+          string_of_int m
+      | _ -> "")
+  | "death_place" -> (
+      match c with
+      | Update.Create (_, Some { ci_death_place = pl }) ->
+          safe_val (Util.escape_html pl :> Adef.safe_string)
+      | _ -> str_val "")
+  | "death_year" -> (
+      str_val
+      @@
+      match c with
+      | Update.Create (_, Some { ci_death_date = Some (Dgreg (dmy, Dfrench)) })
+        ->
+          let dmy = Calendar.french_of_gregorian dmy in
+          add_precision (string_of_int dmy.year) dmy.prec
+      | Update.Create
+          (_, Some { ci_death_date = Some (Dgreg ({ year = y; prec = p }, _)) })
+        ->
+          add_precision (string_of_int y) p
+      | Update.Create (_, Some { ci_death = death; ci_death_date = None }) -> (
+          match death with DeadDontKnowWhen -> "+" | NotDead -> "-" | _ -> "")
+      | _ -> "")
+  | "occupation" -> (
+      match c with
+      | Update.Create (_, Some { ci_occupation = occupation }) ->
+          safe_val (Util.escape_html occupation :> Adef.safe_string)
+      | _ -> str_val "")
+  | "sex" -> (
+      str_val
+      @@
+      match c with
+      | Update.Create (Male, _) -> "male"
+      | Update.Create (Female, _) -> "female"
+      | Update.Create (Neuter, _) -> "neuter"
+      | _ -> "")
+  | _ -> raise Not_found
+
+and add_precision s p =
+  match p with
+  | Maybe -> "?" ^ s
+  | Before -> "&lt;" ^ s
+  | After -> "&gt;" ^ s
+  | About -> "/" ^ s ^ "/"
+  | Sure | OrYear _ | YearInt _ -> s
+
```

Comment:
was this in updateFam.ml ?
you can try to put shared code in update_util.ml

Reply:
j'allais le faire.
Petit ajout dans updateIndOk.ml pour les relations après avoir reporté cette modif sur la 6.08 de Roglo.
Pour l'instant, seul le statut "public" est pris en compte. 
Les dates de naissance/décès sont probablement rarement disponibles, par contre "occupation" pourrait être avantageusement enregistré à ce moment.
Qu'en pensez vous?